static/templates/stream_settings/stream_types.hbs

```diff
@@ -27,6 +27,15 @@
     </div>
 {{/if}}
 
+<div id="default-stream">
+    <label class="checkbox">
+        <input type="checkbox" name="default_stream" value="default_stream" class="prop-element" id="id_is_default_stream" data-setting-widget-type="boolean" {{#if check_default_stream}}checked{{/if}}/>
+        <span></span>
+        {{t "Default stream for new users" }}
+        {{> ../help_link_widget link="/help/set-default-streams-for-new-users" }}
+    </label>
+</div>
```

Comment:
Can we use already existing `settings_checkbox` or `stream_settings_checkbox` template here?

Reply:
I think we can use `settings_checkbox` template but needs to add the id/class to the container for styling. Updated the PR for the same.